Domain.Supply/Entities/SupplyRequirementEntity.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Diagnosea.Submarine.Domain.Supply.Entities
+{
+    public class SupplyRequirementEntity
```

Comment:
Going to have to give the Compatibility/Requirement concept some serious thought. How can you compare compatibility and requirement when you have varying metrics to define dosage/amount/value.